plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -229,7 +229,7 @@ export class EventsProcessor {
             topic: this.pluginsServer.CLICKHOUSE_JSON_EVENTS_KAFKA_TOPIC,
             key: uuid,
             value: Buffer.from(JSON.stringify(rawEvent)),
-            waitForAck: true,
+            waitForAck: false,
```

Comment:
Still working on this. The hardcoded `true` seemed wrong to me here, but I think I misunderstood.

The issue (and reason I got here) is that I'm getting `catch is not a function` on what we expect to be `Promise` objects...